src/blocks/mod.rs

```diff
@@ -1,49 +1,52 @@
-mod time;
```

Comment:
I think that the changes to this file are unintentional format changes. Can you remove changes that are not relevant to the new block?

Reply:
Yes, I sorted them in order to get cleaner looking code. Should I just remove it, or is a pull request welcome?